mltrace/entities/history.py

```diff
@@ -0,0 +1,73 @@
+
+
+import copy
+from datetime import datetime
+
+from mltrace.db import Store
+from mltrace import utils as clientUtils
+from mltrace.entities import base
+
+
+class History(object):
+    def __init__(
+        self,
+        componentName,
+    ):
+        self.component_name = componentName
+
+    def get_runs_by_time(
+        self,
+        start_time: datetime,
+        end_time: datetime,
+    ):
+        store = Store(clientUtils.get_db_uri())
+        history_runs = store.get_history(
+            self.component_name, None, start_time, end_time)
+        history_runs = History.convertToClient(history_runs)
+        return history_runs
+
+    def get_runs_by_index(
+        self,
+        front_idx: int,
+        last_idx: int,
+    ):
+        store = Store(clientUtils.get_db_uri())
+        history_runs = store.get_component_runs_by_index(
+            self.component_name, front_idx, last_idx)
+        history_runs = History.convertToClient(history_runs)
+        return history_runs
+
+    # helper function - convert to client facing component runs
+    def convertToClient(componentRuns):
```

Comment:
Put this function in `mltrace/utils.py` so you can avoid the import to `base` and import the helper function directly from `mltrace/utils.py`.